### ISSUE TITLE
How to get the run.properties files for storm_track_gen.pl code?

### ISSUE THREAD

**Issue by jalal-90** [StormSurgeLive/asgs#1174]
I am trying to run the storm_track_gen.pl code to get the fort.22 file for ADCIRC model. But I am getting an error that nhcConsensus: storm_track_gen.pl: Failed to open run.properties: No such file or directory. 

Any suggestions would be appreciated. 

![Weixin Image_20230801105410](https://github.com/StormSurgeLive/asgs/assets/139944952/d3086253-c8c4-4d79-83fe-9ad9ddd95318)

**Comment by jasonfleming** [comment]
This is related to #1031 ... we are in the process of fixing this. For now, I think you can comment out the code that loads the `run.properties` file.

**Comment by jalal-90** [comment]
Thank you very much, sir, for your kind advice. I added comment for the code that loads the run.properties file. And I use the following command:
perl storm_track_gen.pl --dir ./ --storm 09 --year 2021 --startdate 2021071900 --name IN-FA

However, I am getting the following errors:
Unknown option: startdate
[2023-Aug-2-T11:30:08] INFO: IN-FA: storm_track_gen.pl: The cold start date was not specified using the --coldstartdate argument. The date/time of the most recent hindcast is '2021072606'. This will be used as the coldstart date/time.
[2023-Aug-2-T11:30:08] ERROR: IN-FA: storm_track_gen.pl: Failed to open forecast ATCF file .//wp092021.fst for ensemble member 'IN-FA': No such file or directory.
Died at storm_track_gen.pl line 245.

For your kind consideration, I added the modified "storm_track_gen.pl" code and the best track data in the attachment.

I am waiting for your kind advice.

[code_data.zip](https://github.com/StormSurgeLive/asgs/files/12235423/code_data.zip)

![Weixin Image_20230802113803](https://github.com/StormSurgeLive/asgs/assets/139944952/73c28c3b-057a-4e23-84bd-383642bdef29)
